backend/app2/api/routes/assistant_streaming.py

```
@@ -342,33 +348,49 @@ async def process_generate_request(
 
     # Generate music with music_gen_service, passing the session
     try:
-        response = await music_gen_service.compose_music(
-            context.prompt, sse_queue, session
+        # response = await music_gen_service.compose_music(
+        #     context.prompt, sse_queue, session
+        # )
+        selected_model_info = available_models.get_model_by_name(context.model)
+        selected_model_info.api_key = selected_model_info.get_api_key()
+        logger.info(f"Selected model: {selected_model_info}")
+        response = await music_agent.run(
```

Ruff (0.8.2)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)